SU-SWS/stanford_paragraph_types
# Earth 138

# READY FOR REVIEW

# Summary
- Changes component template to a centered one.
- Changed the image style to the correct scale and crop size.

# Needed By (Date)
- End of sprint.

# Urgency
- Moderate

# Steps to Test

1. Check out this branch
2. Check out EARTH-138 branch in stanford_components
3. Import feature `drush feature-import stanford_paragraph_cta_list` & `drush feature-import stanford_paragraph_cta`
4. Clear caches
5. Create a CTA list paragraph and review the image style and centered template

# Affected Projects or Products
- Matson theme has a PR
- Stanford Components has a PR

# Associated Issues and/or People
- EARTH-138

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
